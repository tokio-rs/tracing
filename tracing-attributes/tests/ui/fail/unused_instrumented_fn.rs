#![deny(dead_code)]

#[tracing::instrument]
fn never_used() {}

fn main() {}
