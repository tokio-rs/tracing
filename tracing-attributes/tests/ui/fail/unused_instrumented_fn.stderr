error: function `never_used` is never used
 --> tests/ui/fail/unused_instrumented_fn.rs:4:4
  |
4 | fn never_used() {}
  |    ^^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/ui/fail/unused_instrumented_fn.rs:1:9
  |
1 | #![deny(dead_code)]
  |         ^^^^^^^^^
